lib/trino-parquet/src/main/java/io/trino/parquet/writer/PrimitiveColumnWriter.java

```diff
@@ -241,7 +246,7 @@ private void flushCurrentPageToBuffer()
         repetitionLevelWriter.reset();
         definitionLevelWriter.reset();
         primitiveValueWriter.reset();
-        updateBufferedBytes();
+        updateBufferedBytes(getCurrentPageBufferedBytes());
```

Comment:
isn't `getCurrentPageBufferedBytes()` 0 at this point?

Reply:
it might not be 0 due to dictionary which gets flushed at the end of row group